ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/defs_application_traffic_recognition.schema.yml

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+$defs:
+  application_traffic_recognition_application:
+    type: list
+    items:
```

Comment:
Since the def is really for a single application, I think we should move the ref to the inner dict level.
```suggestion
```